### PR TITLE
First pull request

### PR DESCRIPTION
**Pull request by DarthHater**
This is mainly for teaching purposes, but this is a lil pull request that adds:

- A tiny lil golang app, which we will build on
- LICENSE file, so that consumers of this project know their legal rights to using it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
 # locquacious-langosta :lobster:
 ### Don't get stuck talking to either or the tide will take you away. I spoke to a cephalopod once for 16 hours and lost my shoes. I got kinda crabby :crab:. Luckily, it was during a Neap tide when the tidal range is at its smallest. 
+
+### Development Instructions
+
+This project now has a lil golang app included in it. To get it running:
+
+- Install golang (1.16 or higher is a good choice)
+
+In the root, run `go run main.go`
```
Comment (vampyteuth) [inline README.md]: Could you please refer to this as the project root.

**Review by vampyteuth — CHANGES_REQUESTED** [review]
I've requested a change.

File: README.md (hunk 1 above)
Comment (DarthHater) [reply README.md]: You got it!

Comment (DarthHater) [reply README.md]: Change done in: 8edfee2

**Review by vampyteuth — APPROVED** [review]

**Pull request merged**